### DISCUSSION TITLE
How to customize a theme?

### DISCUSSION THREAD

**Discussion by biapar** [carbon-design-system/carbon#12699] [Help]
Hi, I wish to customize the primary button color in the themes.
I don't found an example or a single main guide. 
Which are the methods for Carbon v11 on React or on Javascript ?
Thanks.

**Comment by tw15egan** [comment]
I would start by taking a look here: 
https://carbondesignsystem.com/guidelines/themes/overview#default-theme

and here:
https://github.com/carbon-design-system/carbon/tree/main/packages/themes#usage


If you are trying to theme button tokens, you'll also need to pull in the button tokens:

`@use '@carbon/styles/scss/components/button/tokens';`

More info: 
https://github.com/carbon-design-system/carbon/blob/main/packages/styles/docs/sass.md#components (scroll down to component tokens)


If you have the repo installed locally, you can also `cd packages/examples/custom-theme` and run the example app that shows how to implement custom theming. Here is the [source code](https://github.com/carbon-design-system/carbon/tree/main/examples/custom-theme).

**Reply by biapar** [reply]
For example I wish only to change the color of $background-brand or the primary button.
How to?

**Comment by biapar** [comment]
This works...https://github.com/carbon-design-system/carbon/discussions/11313#discussioncomment-3519672

https://github.com/carbon-design-system/carbon/blob/main/packages/styles/docs/sass.md#components

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/discussions/11313#discussioncomment-3519672
